practical/groovy/idr0021.groovy

```diff
@@ -80,15 +86,7 @@ import ij.plugin.frame.RoiManager
 import ij.measure.ResultsTable
 
 
-// Setup
-// =====
-
-// OMERO Server details
-HOST = "workshop.openmicroscopy.org"
-PORT = 4064
 project_id = 1101
```

Comment:
```suggestion

```

This line seems to have overriden whatever I have put into the dialog.

Reply:
Needs to be removed or somehow fixed - not sure about the implications in further code, but at present, this is not working as is.